cmdutils/switches/switches_test.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021 OnMetal authors
```

Comment:
Tests are missing for
* `Switches.All`
* `Switches.DisabledByDefault`
* `Switches.String`

Additionally, could you also please add a test using `flag` / `pflag`? Often, through the usage of default values etc. there, there are some problems. There, we should have the following scenarios:
* Define `Switches` with some default / disabled by default values, then
  * Have a test that only enables a list of specific controllers, all others should be disabled like `enable-1,enable-2`
  * Have a test that uses the default set by passing in `*` but also disables some of them, so the total string would look sth like `*,-disable-1,-disable-2`
  * Have a test that enables an unknown controller (should error)
  * Have  a tests that does sth like `*,enable-sth-not-enabled-by-default,-disable-sth-enabled-by-default`

Reply:
Added both interface and package-related(`flag`, `pflag`) tests.